recipes/icc/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class ICCConan(ConanFile):
+    name = 'icc'
```

Comment:
```suggestion
    name = 'inter-component-communication '
```

ICC is too generic and that project is still not very popular. A quick search on Github shows me more C/C++ projects using same name, even more popular (more forks, more stars): https://github.com/search?l=C%2B%2B&q=icc&type=Repositories, for instance: https://github.com/snumrl/ICC

Also, there is a rule about name collision for Conan Center Index:

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-on-recipe-name-collisions

Reply:
It is to long name to type in console ...
Any other suggestions ?

Reply:
cargo (Rust package manager) also has such issue, but it allow to specify user as well as conan

Reply:
Sustainability > convince... we can not change the name after 

Perhaps

```suggestion
    name = 'redradist-icc'
```

Reply:
@prince-chrismc I think maybe better `icc-redradist` ...

Reply:
> 
> 
> Sustainability > convince... we can not change the name after
> 
> Perhaps

@prince-chrismc But we can specify package and channel ... like this icc/1.0.0@redradist

Reply:
We can go with `redradist-icc` 👍 


> @prince-chrismc But we can specify package and channel ... like this icc/1.0.0@redradist

No. https://github.com/conan-io/conan-center-index/blob/c7b9c1b9a0a2b8e9ae9fccdfeec55a91728d0ae0/docs/how_to_add_packages.md#more-information-about-recipes

Reply:
Seems like it happens only due to removing support @user/channel ...
Cargo allows to have multiple packages with the same name provided by different maintainers even with different functionality ...

`redradist-icc` or `icc-redradist` ?
Also according instruction I would need to rename receipt folder like `redradist-icc` ?

Reply:
Done, renamed